src/debugger/com.c

```diff
@@ -453,13 +453,13 @@ int xdebug_is_debug_connection_active()
 	);
 }
 
-int xdebug_is_debug_connection_active_for_current_pid()
+void xdebug_activate_connection_for_current_pid()
```

Comment:
The `base` code shouldn't really be calling this directly, and only code from within the `src/debugger` directory should. Instead, there should be an API function in `src/debugger/debugger.c` that calls this. Perhaps called `xdebug_debugger_restart_if_pid_changed`.